sdk/core/abort-controller/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.4 (Unreleased)
 
+Fixes issue [13985](https://github.com/Azure/azure-sdk-for-js/issues/13985) where abort event listeners that removed themselves when invoked
+could prevent other event listeners from being invoked.
```

Comment:
can we put this onto one line and let the md wrap itself?

Reply:
I have removed the manual word-wrapping!